packages/babel-generator/test/fixtures/comments/variable-declarator-trailing-comment/output.js

```diff
@@ -1,20 +1,19 @@
 {
-  var t = 20;
-  /*
-   * This is trailing comment
-   */
+  var t = 20; /*
+             * This is trailing comment
+             */
```

Comment:
The indentation here looks off :thinking: Is it possible to align the `*`s?

Reply:
It seems to have exposed two previous issues, one from a regression from the last time I improved performance, and one that was always there, and now they're both fixed!

Also I found out that indenting `var` affects `leadingComments`.
Now the `VariableDeclarator` is no longer aligned, but indented by 2 spaces.
Now the indentation of `var a={x},b={y}` has been greatly improved, which I think looks good.

![image](https://user-images.githubusercontent.com/30521560/193621196-022db951-cff1-4bc4-a5ec-9f45895830a2.png)
![image](https://user-images.githubusercontent.com/30521560/193621238-01e80c33-ed8e-4443-97a3-23a2302d21e9.png)
![image](https://user-images.githubusercontent.com/30521560/193621314-7aa2b925-cbe5-47f8-8124-a60a8ad711a6.png)
![image](https://user-images.githubusercontent.com/30521560/193621449-9cff82a2-0c2e-4241-bcef-cd5d8e1efa4e.png)